owl/SOMA.owl

```diff
@@ -148,7 +154,25 @@ AnnotationPropertyRange(SOMA:symbol xsd:string)
 #   Object Properties
 ############################
 
-# Object Property: <http://www.ease-crc.org/ont/SOMA.owl#after> (<http://www.ease-crc.org/ont/SOMA.owl#after>)
+# Object Property: <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> (is initial scene of)
```

Comment:
I'm somewhat surprised that this file has this much content, given the initial annotation about how it is to be used: as a list of common imports, and that any axioms in it will eventually be moved to the specialized modules. I think then that the definitions related to StateTransitions that we see here should be in SOMA-STATE or SOMA-ACT.

Reply:
I agree, but would prefer not to solve this in this PR but in a separate issue - we have to revise the import structure maybe a little bit, because some ontologies use entities that are not imported and fly around loosely.